src/main/resources/application.yaml

```diff
@@ -23,17 +23,22 @@ spring:
     properties:
       charSet: UTF-8
     # http://tomcat.apache.org/tomcat-7.0-doc/jdbc-pool.html#Common_Attributes
+    max-active: 10
     tomcat:
       max-active: 10
```

Comment:
Do we want max-active within namespace and outside namespace?

Reply:
good point, i missed that. let me check that wasn't the thing that fixed it!